report.Rmd

```diff
@@ -10,6 +10,7 @@ params:
     language links that are similar to the individual search results. A
     control group will see the currently existing search results page.
   data: "./data/Explore similar pages, categories and suggested languages in search results/events_raw_20170724112938.RData"
+  query: "SELECT * FROM TestSearchSatisfaction2_16909631;"
```

Comment:
If I don't read params from an external yaml file and just render report.rmd, it works. But if I do `query:` or `query: NULL` in rmd file, and then run `Rscript run.R` (it use **parameters.yaml** by default, in which `query: NULL`), I will get this error message:
```
Error in knit_params_get(input_lines, params) : 
  render params not declared in YAML: query
```